mappings/net/minecraft/network/listener/ClientPlayPacketListener.mapping

```diff
@@ -205,3 +205,5 @@ CLASS net/minecraft/class_2602 net/minecraft/network/listener/ClientPlayPacketLi
 		ARG 1 packet
 	METHOD method_44075 onServerMetadata (Lnet/minecraft/class_7495;)V
 		ARG 1 packet
+	METHOD method_44286 onChatPreviewEnabled (Lnet/minecraft/class_7519;)V
```

Comment:
Rename?

Reply:
Fixed in 8c0672854874e8c9aae52a07e8508c73a763e814